boards/same54-xpro/Makefile.features

```diff
@@ -16,6 +16,7 @@ FEATURES_PROVIDED += periph_uart_hw_fc
 FEATURES_PROVIDED += periph_adc
 FEATURES_PROVIDED += periph_usbdev
 FEATURES_PROVIDED += periph_freqm
+FEATURES_PROVIDED += periph_can
```

Comment:
this is in the wrong commit

Reply:
Elaborate please. Do you want me to remove that one line of code to another commit?

Reply:
Or do you want to make commits per file?

Reply:
this change is in the cpu/sam.. commit  but the change that this feature is describing is happening in the board/sam54.. commit  -> this is the wrong commit

Reply:
Solution I offer -> I will collect the cpu touching commits in one and the board ones in another commit -> hopefully no wrong commits

Reply:
good for me,

for the test either have one commit for the test or have it completly with the board ( no updatecommit)

please also have no `update` / `fix` / or / `ci`  comment within your commit message

Reply:
Done